app/models/framework/management_charge_calculator.rb

```diff
@@ -0,0 +1,8 @@
+class Framework
+  module ManagementChargeCalculator
+    Dir['app/models/framework/management_charge_calculator/*'].each do |definition|
```

Comment:
I am a bit lost as to what is happening here and why. Could you amend the commit message perhaps with a brief explanation?

Reply:
Will do. I'm loading all of the ManagementChargeCalculator classes in here, similar to https://github.com/dxw/DataSubmissionServiceAPI/blob/develop/app/models/framework/definition.rb#L53-L59

Reply:
@CristinaRO Actually, the autoloader does this, so I'm not sure what I was thinking. Removed.

Reply:
Actually, I removed it and it still worked, so 🤦‍♂️